streaming/eslint.config.mjs

```diff
@@ -0,0 +1,38 @@
+// @ts-check
+
+import globals from 'globals';
+import tseslint from 'typescript-eslint';
+
+import { baseConfig } from '../eslint.config.mjs';
+
+export default tseslint.config([
+  baseConfig,
+  {
+    languageOptions: {
+      globals: globals.node,
+
+      ecmaVersion: 2021,
+      sourceType: 'module',
+    },
+
+    rules: {
+      'no-delete-var': 'off',
+
+      'import/no-extraneous-dependencies': [
+        'error',
+        {
+          devDependencies: ['eslint.config.mjs'],
+          optionalDependencies: false,
+          peerDependencies: false,
+          includeTypes: true,
+          packageDir: import.meta.dirname,
+        },
+      ],
+
+      'import/extensions': ['error', 'always'],
+
+      // TODO: Fix resolution of imports
```

Comment:
We should probably fix this? I am not sure it would be hard to do so

Reply:
looks like it's fine now.